doc/developer-guides/hld/hv-cpu-virt.rst

```diff
@@ -895,6 +895,10 @@ pass-through directly:
      - TSC target of local APIC's TSC deadline mode
      - emulates with vlapic
 
+   * - MSR_IA32_TSC_ADJUST
+     - TSC adjustment of local APIC's TSC deadline mode
+     - emulates with vlapic
+
```

Comment:
It's better to move the MSR_IA32_TSC_ADJUST ahead to keep the same order of MSRs as the source code.